enterprise/server/remote_execution/runner/runner.go

```diff
@@ -140,6 +140,9 @@ var (
 )
 
 func k8sPodID() (string, error) {
+	if podID := os.Getenv("K8S_POD_ID"); podID != "" {
```

Comment:
could you put this in `server/resources/resources.go` and add a getter like `GetK8sPodID()` just so all the env vars are in one place (MY_POOL, MY_NODENAME, MY_HOSTNAME etc.)